group_3/script.R

```diff
@@ -0,0 +1,122 @@
+# Group 3
+# Matthew Bodziony, Daniel Choi, Cole Groeneveld
+
+# Path to Git Bash on your computer
+git_bash_path <- "C:/Program Files/Git/git-bash.exe"
+
+# Write your bash commands and run them  
+curl_command <- "curl -L -o ./raw_data/archive.zip https://www.kaggle.com/api/v1/datasets/download/tylermorse/retail-business-sales-20172019"
+system(paste(shQuote(git_bash_path), 
+             "-c", 
+             shQuote(curl_command)), 
+       intern = TRUE)
+
+unzip_command <- "unzip -o ./raw_data/archive.zip -d ./business_sales"
+system(paste(shQuote(git_bash_path), 
+             "-c", 
+             shQuote(unzip_command)), 
+       intern = TRUE)
+
+# Load necessary package
+library(readr)
+
+# Load the dataset
+data_frame_name <- read_csv("business_sales/business.retailsales.csv")
+
+# Confirm data is loaded
+print(head(data_frame_name))
+
+# Load additional libraries
+library(dplyr)
+library(skimr)
+
+# Initial inspection of the dataset
+glimpse(data_frame_name)
+head(data_frame_name)
+tail(data_frame_name)
+skim(data_frame_name)
+
+# Document notable observations
+# - The dataset contains 1775 rows and 6 columns
+# - The columns contain "Product Type, Net Quantity, Gross Sales, 
+# - Discounts, Returns, and Total Net Sales".
+# - The Rows contain different product types.
+# - There seems to be 8 missing product types which leads to a 99.5% completion rate
+
+# Key columns: Product Type and Total Net Sales
+selected_data <- data_frame_name %>%
+  select(`Product Type`,`Total Net Sales`)
```

Comment:
how about making the column names neat like `product_type` with the janitor package?